conans/client/command.py

```diff
@@ -1267,7 +1267,11 @@ def run(self, *args):
         """HIDDEN: entry point for executing commands, dispatcher to class
         methods
         """
-        errors = False
+        SUCCESS = 0
+        ERROR_GENERAL = 1
+        ERROR_INVALID_CONFIGURATION = 5
+
```

Comment:
I think that we need a single place to put this hardcoded values, as they are tightly coupled with those documented in the `main` method:

```python
def main(args):
    """ main entry point of the conan application, using a Command to
    parse parameters

    Exit codes for conan command:

        0: Success (done)
        1: General ConanException error (done)
        2: Migration error
        3: Ctrl+C
        4: Ctrl+Break
        5: Invalid configuration (done)
    """
```

Reply:
Yes, feel free to move them to global scope, I guess here, in the ``command.py`` file? Other file?